### DISCUSSION TITLE
JSON

### DISCUSSION THREAD

**Discussion by yeoffrey** [catppuccin/catppuccin#2267] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

I've looked and can't find a basic JSON file that just lists every schema with the hex codes as a JSON object. Would be useful for any language or development tool. I have a version of it but not sure if its something that belongs here.

This would then make sense to implement in TOML, YAML, or Pkl too.

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by nekowinston** [comment]
https://github.com/catppuccin/palette/blob/main/palette.json is the single source of truth for lots of our libraries (npm, deno, rust, go, etc.)

**Reply by yeoffrey** [reply]
Thank you very much!

**Reply by yeoffrey** [reply]
You might consider linking to this file in the main page under development tools or libraries. If so, I am happy to make the PR for that.

**Reply by sgoudham** [reply]
Hey, we do have the palette repository linked under the [Palette](https://github.com/catppuccin/catppuccin?tab=readme-ov-file#-palette) section so I don't think linking to the file makes sense there.

However, it might be worth explicitly calling out the file in the README in catppuccin/palette?

**Reply by yeoffrey** [reply]
Thanks for the link! I think linking directly in `catppuccin/palette` would be nice.

Edit: I'm happy to make that PR as well still 👍
